corehq/apps/app_manager/models.py

```diff
@@ -2226,6 +2270,88 @@ def update_app_case_meta(self, app_case_meta):
                 meta.add_closer(self.unique_id, action.close_condition)
 
 
+class SchedulePhaseForm(IndexedSchema):
+    """
+    A reference to a form in a schedule phase.
+    """
+    form_id = FormIdProperty("modules[*].schedule_phases[*].forms[*].form_id")
+
+
+class SchedulePhase(IndexedSchema):
+    """
+    SchedulePhases are attached to a module.
+    A Schedule Phase is a grouping of forms that occur within a period and share an anchor
+    A module should not have more than one SchedulePhase with the same anchor
+
+    anchor:                     Case property containing a date after which this phase becomes active
+    forms: 			The forms that are to be filled out within this phase
+    """
+    anchor = StringProperty()
+    forms = SchemaListProperty(SchedulePhaseForm)
+
+    @property
+    def id(self):
+        """ A Schedule Phase is 1-indexed """
```

Comment:
When I started building this I thought that phase 0 would be referenced when the case had not entered into a schedule. This ended up being unnecessary since I check current_schedule_phase = '' to be the 0th phase, but I never changed this back. I'm going to leave it this way for now since it matches the numbering on the edit phases page (and 0-based indexing is weird when it is user-facing)